style.css

```diff
@@ -54,11 +65,130 @@ a:visited {
 }
 
 .product-container .title {
+  font-family: 'Open Sans', sans-serif;
   padding-bottom: 28px;
   font-weight: 700;
   text-transform: uppercase;
 }
 
 .product-container .tagline {
+  padding-bottom: 12px;
+  font-family: 'Dosis', sans-serif;
   font-style: italic;
 }
+
+/* Product Overlay Description */
+
+.overlay {
+  position: absolute;
+  z-index: 3;
+  height: 100%;
+  padding: 10px;
+  overflow: hidden;
+
+  color: #F8F8F8;
+  background: #2D2D2D;
+
+  font-family: 'Dosis', sans-serif;
+  font-size: 16px;
+  font-weight: 100;
+  line-height: normal;
+  letter-spacing: 1px;
+
+  opacity: 0;
+  transition: opacity .50s ease;
+   -moz-transition: opacity .50s ease;
+   -webkit-transition: opacity .50s ease;
+}
+
+.overlay:hover {
+  opacity: 0.9;
+}
+
+/* Remove Button */
+
+.action-remove {
+  float: right;
+  z-index: 5;
+  width: 18px;
+  padding: 3px;
+  margin: -10px -7px 0 0;
+
+  color: white;
+  font-size: 16px;
+  font-weight: bold;
+
+  opacity: 0.3;
+  transition: opacity .30s ease;
+   -moz-transition: opacity .30s ease;
+   -webkit-transition: opacity .30s ease;
+}
+
+.action-remove:hover {
+  opacity: 1;
+  color: white;
+}
+
+/* Animation for Removing Product */
+
+@-webkit-keyframes tvOut {
+  0% {
+    opacity: 1;
+  }
+
+  50% {
+    opacity: 0;
+    -webkit-transform: scale3d(.9, .1, .3);
+    transform: scale3d(.9, .1, .3);
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+@keyframes tvOut {
+  0% {
+    opacity: 1;
+  }
+
+  50% {
+    opacity: 0;
+    -webkit-transform: scale3d(.9, .1, .3);
+    transform: scale3d(.9, .1, .3);
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+.tvOut {
+  -webkit-animation-duration: 0.5s;
+  animation-duration: 0.5s;
+  -webkit-animation-name: tvOut;
+  animation-name: tvOut;
+}
+
+
+/* Responsive Typography */
+
+@media (min-width:767px) and (max-width: 991px) {
+  .page-title {
+    font-size: 32px;
+  }
+}
+
+@media (min-width:992px) and (max-width: 1200px) {
+  .overlay {
+    font-size: 13px;
+  }
+}
+
+@media (max-width: 500px) {
+  .page-title {
+    font-size: 26px;
+  }
+}
+
+
```

Comment:
Dynamically change typography depending on different viewport size to improve the app's structure and readability.